Error: The terraform-provider-grafana_v1.29.0 plugin crashed!

### Terraform Version
* Terraform: ... 1.3.1 
* Terraform Grafana Provider: ... 1.29.0 
* Grafana: ... Grafana Cloud (latest) 

### Affected Resource(s)
Please list the resources as a list, for example:
* grafana_alert

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files

```
variable "email_addresses" {
  type        = list(any)
  description = "Email addresses"
  default     = [""]
}
```

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

```
module.grafana_test_service_slack.grafana_contact_point.slack_contact_point: Modifying... [id=tp3t_fVVk]
╷
│ Error: Plugin did not respond
│ 
│   with module.grafana_test_service_slack.grafana_contact_point.slack_contact_point,
│   on ../modules/contact_points/slack/main.tf line 2, in resource "grafana_contact_point" "slack_contact_point":
│    2: resource "grafana_contact_point" "slack_contact_point" {
│ 
│ The plugin encountered an error, and failed to respond to the
│ plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may
│ contain more details.
╵

Stack trace from the terraform-provider-grafana_v1.29.0 plugin:

panic: interface conversion: interface {} is nil, not string

goroutine 69 [running]:
github.com/grafana/terraform-provider-grafana/grafana.unpackAddrs({0xc000531ec0, 0x1, 0xdc3070?})
	github.com/grafana/terraform-provider-grafana/grafana/resource_alerting_contact_point_notifiers.go:341 +0x157
github.com/grafana/terraform-provider-grafana/grafana.emailNotifier.unpack({}, {0xcc70a0?, 0xc0004fe570}, {0xc00047ad14, 0x0?})
	github.com/grafana/terraform-provider-grafana/grafana/resource_alerting_contact_point_notifiers.go:311 +0x199
github.com/grafana/terraform-provider-grafana/grafana.unpackPointConfig({0xf1cac0?, 0x14b7660?}, {0xcc70a0?, 0xc0004fe570?}, {0xc00047ad14?, 0xc0005ccb70?})
	github.com/grafana/terraform-provider-grafana/grafana/resource_alerting_contact_point.go:222 +0x89
github.com/grafana/terraform-provider-grafana/grafana.unpackContactPoints(0xc00047ad02?)
	github.com/grafana/terraform-provider-grafana/grafana/resource_alerting_contact_point.go:213 +0x2cf
github.com/grafana/terraform-provider-grafana/grafana.updateContactPoint({0xf1c388, 0xc000114a80}, 0xc0000dd800, {0xc64240?, 0xc000300420?})
	github.com/grafana/terraform-provider-grafana/grafana/resource_alerting_contact_point.go:154 +0x1d8
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).update(0xc0003d6540, {0xf1c3c0, 0xc0002991d0}, 0xd?, {0xc64240, 0xc000300420})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.23.0/helper/schema/resource.go:741 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc0003d6540, {0xf1c3c0, 0xc0002991d0}, 0xc000285a00, 0xc0000dd680, {0xc64240, 0xc000300420})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.23.0/helper/schema/resource.go:847 +0x82c
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc00030f218, {0xf1c3c0?, 0xc0002990b0?}, 0xc000726dc0)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.23.0/helper/schema/grpc_provider.go:1021 +0xe3c
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc000364dc0, {0xf1c3c0?, 0xc0002988a0?}, 0xc0001ba540)
	github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov5/tf5server/server.go:818 +0x574
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0xd8b660?, 0xc000364dc0}, {0xf1c3c0, 0xc0002988a0}, 0xc0001ba3f0, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:385 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00046a000, {0xf1f460, 0xc000498000}, 0xc0005f86c0, 0xc00046c7e0, 0x1476e00, 0x0)
	google.golang.org/grpc@v1.48.0/server.go:1295 +0xb0b
google.golang.org/grpc.(*Server).handleStream(0xc00046a000, {0xf1f460, 0xc000498000}, 0xc0005f86c0, 0x0)
	google.golang.org/grpc@v1.48.0/server.go:1636 +0xa1b
google.golang.org/grpc.(*Server).serveStreams.func1.2()
	google.golang.org/grpc@v1.48.0/server.go:932 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.48.0/server.go:930 +0x28a

Error: The terraform-provider-grafana_v1.29.0 plugin crashed!
```

### Expected Behavior
What should have happened? 

### Actual Behavior
What actually happened?

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

--- a/grafana/adapters.go
+++ b/grafana/adapters.go
@@ -7,7 +7,11 @@ import (
 func listToStringSlice(src []interface{}) []string {
 	dst := make([]string, 0, len(src))
 	for _, s := range src {
-		dst = append(dst, s.(string))
+		val, ok := s.(string)
+		if !ok {
+			val = ""
+		}
+		dst = append(dst, val)
 	}
 	return dst
 }
@@ -19,7 +23,11 @@ func setToStringSlice(src *schema.Set) []string {
 func listToIntSlice(src []interface{}) []int {
 	dst := make([]int, 0, len(src))
 	for _, s := range src {
-		dst = append(dst, s.(int))
+		val, ok := s.(int)
+		if !ok {
+			val = 0
+		}
+		dst = append(dst, val)
 	}
 	return dst
 }

--- a/grafana/resource_alerting_contact_point_notifiers.go
+++ b/grafana/resource_alerting_contact_point_notifiers.go
@@ -7,6 +7,7 @@ import (
 
 	gapi "github.com/grafana/grafana-api-golang-client"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
 type alertmanagerNotifier struct{}
@@ -258,7 +259,8 @@ func (e emailNotifier) schema() *schema.Resource {
 		Required:    true,
 		Description: "The addresses to send emails to.",
 		Elem: &schema.Schema{
-			Type: schema.TypeString,
+			Type:         schema.TypeString,
+			ValidateFunc: validation.StringIsNotEmpty,
 		},
 	}
 	r.Schema["single_email"] = &schema.Schema{
@@ -336,10 +338,7 @@ func packAddrs(addrs string) []string {
 }
 
 func unpackAddrs(addrs []interface{}) string {
-	strs := make([]string, 0, len(addrs))
-	for _, addr := range addrs {
-		strs = append(strs, addr.(string))
-	}
+	strs := listToStringSlice(addrs)
 	return strings.Join(strs, addrSeparator)
 }
 

--- a/grafana/resource_alerting_notification_policy.go
+++ b/grafana/resource_alerting_notification_policy.go
@@ -7,6 +7,7 @@ import (
 	gapi "github.com/grafana/grafana-api-golang-client"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
 func ResourceNotificationPolicy() *schema.Resource {
@@ -40,7 +41,8 @@ This resource requires Grafana 9.1.0 or later.
 				Required:    true,
 				Description: "A list of alert labels to group alerts into notifications by. Use the special label `...` to group alerts by all labels, effectively disabling grouping.",
 				Elem: &schema.Schema{
-					Type: schema.TypeString,
+					Type:         schema.TypeString,
+					ValidateFunc: validation.StringIsNotEmpty,
 				},
 			},
 			"group_wait": {

